app/views/organizations/staff/default_pet_tasks/_default_pet_tasks_table.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class=" mb-4 mb-xl-0 gx-3">
```

Comment:
The table has lost its margin, or padding, from the left and right edges. Not a blocking issue, but worth seeing if you can fix so that the left and right margins of the table align with the left and right margins of the filter. 

<img width="1460" alt="image" src="https://github.com/user-attachments/assets/1eaa5fce-f958-44ab-84d4-164b25641aae">

Reply:
@kasugaijin let me know if the recent change is good to merge

Reply:
@ErinClaudio thank you